docs/language.md

```diff
@@ -48,6 +48,15 @@ Note that among the builders that take functions and or values, two types are id
 Making functions part of the type is slightly unusual because the functions passed in could be closures over some variables that exist at that point in the program.
 Right now the implementation doesn't allow this, and we have not tried to formalize what it means.
 
+In addition, it's possible to specify annotations on builder types: these could for example specify an implementation strategy for a builder. To specify an annotation on a `dictmerger`, one can use syntax like
```

Comment:
It might be better to just have a subsection called Annotations and list the ones we support?